src/styles/variables.scss

```diff
@@ -0,0 +1,16 @@
+$calendar-bcg-color: #eee;
+$calendar-gap: 1px;
+$calendar-width: 100px;
+$calendar-height: 100px;
+$calendar-hover-bcg-color: #ffbfcb;
+$start-day: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
```

Comment:
The `$start-day` map keys should follow a consistent naming convention. Consider using a full name for the days of the week to improve readability, for example, `monday` instead of `mon`, `tuesday` instead of `tue`, etc.